src/routes/Search.tsx

```diff
@@ -444,7 +466,15 @@ function SingleContact(props: {
     return (
         <button
             onClick={() => createContactFromSearchResult(props.contact)}
-            class="flex items-center gap-2"
+            class={
```

Comment:
this is a lot of copy pasting so we could probably make a `<ContactButton>` component file to dedupe